### PR TITLE
Finalize blog posts #7

### LINKED ISSUE #39: Fix names of images. `Image_X` is not very descriptive.

### PR DESCRIPTION
**Pull request by etf-yrm**
Finalize blog posts #7 (5/18)
-Add description for correctly visible preview text in category page
-Added missing images in posts
-Replace cover images
-Changed the method of adding images to the standard liquid method
fixes problem #7 

last commit:
fixes problem #39
closes #39

### CODE REVIEW

File: _posts/2019-06-20-why-we-are-doing-what-we-do.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 current: post
-cover:  image_0.png
+cover: image_0.png
 title: Why we are doing what we do
+description: In our inaugural blog post, my aim is to introduce how we see the world and what we care about. Regarding what we do, I will simply say that we are an open source software company concerned with security for cyber-physical systems - i.e. protecting the ‘real world’ stuff that is being connected to the internet like cars, trains, factories, implanted medical devices and of course smart toasters.
 date: 2019-06-20 10:00:00
 category: news
 author: Al Esmail
-tags: [Open Source]
+tags: [engineering,wott,privatelink,heroky postgres,vpc]
```
Comment (vpetersson) [inline _posts/2019-06-20-why-we-are-doing-what-we-do.md]: privatelink? heroky?

File: _posts/2019-06-20-why-we-are-doing-what-we-do.md [hunk 2]
```diff
@@ -24,8 +24,7 @@ The aforementioned examples of internet connected devices are the proverbial tip
 
 At [WoTT]({{site.url}}), we are very excited about this.  We dream of the world of [unmanned aerial vehicles](https://www.airxos.io/) (UAVs) quietly and autonomously zipping around overhead easing congestion and speeding up the time it takes to get from Heathrow to Old street (Newark to the West Village is an equally hateful commute that I would love to solve with UAVs).  We’re patiently waiting for neural implants that will [synthesize speech](https://www.nature.com/articles/s41586-019-1119-1) from thought and that let us skip to the next SNL clip in the queue or adjust the temperature based on how we feel [(assuming Google doesn’t cut access to the necessary APIs)](https://www.home-assistant.io/blog/2019/05/08/nest-data-bye-bye/). If we’re really lucky, maybe we’ll get more transformers - yes, it was done back in 2016.
 
-{% asset blog/neural-implant.png srcset:width="1300 2x" srcset:width="650 1x" alt="No Image" %}
-
+![Representation of a neural implant (FYI - It’s not accurate.)]({{ site.blogimg }}neural-implant.png)
```
Comment (vpetersson) [inline _posts/2019-06-20-why-we-are-doing-what-we-do.md]: Please revert this as discussed.

File: _posts/2019-06-23-why-shift-left-security-is-relevant-for-iot.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
 ---
 layout: post
 current: post
-cover:  image_0.png
+cover:  Image_3.png
```
Comment (vpetersson) [inline _posts/2019-06-23-why-shift-left-security-is-relevant-for-iot.md]: We need to use more descriptive names here.

File: _posts/2019-06-23-why-shift-left-security-is-relevant-for-iot.md [hunk 4]
```diff
@@ -25,8 +26,7 @@ Given this high stakes sport is played by crafty and pragmatic geniuses, the bes
...
 
-{% asset blog/nist.png srcset:width="1300 2x" srcset:width="650 1x" alt="The NIST says so. Prove them wrong if you want" class="img-fluid" %}
-
+![The NIST says so. Prove them wrong if you want]({{ site.blogimg }}nist.png)
```
Comment (vpetersson) [inline _posts/2019-06-23-why-shift-left-security-is-relevant-for-iot.md]: Revert.

File: _posts/2019-06-24-why-open-source-is-critical-for-infosec.md [hunk 5]
```diff
@@ -12,8 +13,7 @@ class: post-template
 
 TL;DR Proprietary security is a fresh cow pie hiding in the tall grass.
 
-{% asset blog/cow.jpg srcset:width="1300 2x" srcset:width="650 1x" alt="Figure 1 - How I imagine decision fatigued CTOs see the world when speaking to proprietary security vendors. Can you spot the cow pie?" class="img-fluid" %}
-
+![How I imagine decision fatigued CTOs see the world when speaking to proprietary security vendors. Can you spot the cow pie?]({{ site.blogimg }}cow.jpg)
```
Comment (vpetersson) [inline _posts/2019-06-24-why-open-source-is-critical-for-infosec.md]: Revert.

File: _posts/2019-06-24-why-open-source-is-critical-for-infosec.md [hunk 6]
```diff
@@ -30,6 +30,8 @@ Let’s begin in 1883 when the Dutch-born cryptographer, Auguste Kerkchoffs arti
...
 
 In searching for academic references to STO, I unfortunately was hard pressed to find even a postulate in favor of the practice.  I did not intend to run STO through the google test but the fourth organic result gives an indication of popular perception (don’t use it).  Though, Ross Patel does make a fair argument that [STO can enhance security](https://www.bcs.org/content/ConWebDoc/2788) in conjunction with other security methods.  He cites the example of network administrators placing sensitive services on non-standard ports to evade script kiddies and programs that exploit common vulnerabilities.  
+![Google test fail3]({{ site.blogimg }}google_test_fail3.png)
```
Comment (vpetersson) [inline _posts/2019-06-24-why-open-source-is-critical-for-infosec.md]: Revert.

File: _posts/2019-06-25-why-open-source-solutions-are-critical-for-IoT.md [hunk 7]
```diff
@@ -12,7 +13,7 @@ class: post-template
 
 TL;DR IoT needs one ring to rule them all and it’s not a platform.
 
-{% asset blog/iot-monster.png srcset:width="1300 2x" srcset:width="650 1x" alt="No Image" class="img-fluid" %}
+![Cyber is sexier than a Harvard trained, Academy Award winning actress.]({{ site.blogimg }}iot-monster.png)
```
Comment (vpetersson) [inline _posts/2019-06-25-why-open-source-solutions-are-critical-for-IoT.md]: Revert.

File: _posts/2019-06-20-why-we-are-doing-what-we-do.md (hunk 1 above)
Comment (etf-yrm) [reply _posts/2019-06-20-why-we-are-doing-what-we-do.md]: This was necessary to verify the correctness of the output array tags.

File: _config.yml [hunk 8]
```diff
@@ -16,11 +16,9 @@ logo: assets/images/blog-icon.png
...
 
-# Social info
-navigation: True # show the navbar links
-subscribers: True # you need to connect an external service for this to work,
-# if 'True' the submit button is disabled for now, but you can change that
-# by editing `_includes/subscribe-form.html`
+#Need add your subscribe mailchimp link list
+mailchimp-list: //redgadgets.us10.list-manage.com/subscribe/post?u=210acce5db69d3d4a04b0e25d&amp;id=08c6708f40
+
 twitter: wott # replace by your username
```
Comment (vpetersson) [inline _config.yml]: This is incorrect.

File: _config.yml [hunk 9]
```diff
@@ -16,11 +16,9 @@ logo: assets/images/blog-icon.png
...
 facebook: wott # replace by your username
```
Comment (vpetersson) [inline _config.yml]: make blank.

File: _config.yml [hunk 10]
```diff
@@ -16,11 +16,9 @@ logo: assets/images/blog-icon.png
 logo_dark: assets/images/favicon.png
 favicon: assets/images/favicon.png
 
-# Social info
-navigation: True # show the navbar links
-subscribers: True # you need to connect an external service for this to work,
-# if 'True' the submit button is disabled for now, but you can change that
-# by editing `_includes/subscribe-form.html`
+#Need add your subscribe mailchimp link list
+mailchimp-list: //redgadgets.us10.list-manage.com/subscribe/post?u=210acce5db69d3d4a04b0e25d&amp;id=08c6708f40
```
Comment (vpetersson) [inline _config.yml]: Should be "https://wott.us20.list-manage.com/subscribe/post?u=250c99a25b42ad842add5bf41&amp;id=ac12838d30"

**Pull request closed without merging**